apiary.apib

```diff
@@ -3217,6 +3217,8 @@ Loads tables from Storage into a Workspace.
                         Default: false
     + preserve (optional, boolean) - Keep existing tables in the workspace, otherwise the workspace will be purged before loading
         Default: false
+    + overwrite (optional, boolean) - When preserve is true duplicate tables will be overwritten
```

Comment:
Pribudla nejaka chybka v dokumentacii. 
![Screenshot 2020-09-22 at 08 20 48](https://user-images.githubusercontent.com/6448364/93849829-31025680-fcad-11ea-9b2d-f731c123540c.png)
![Screenshot 2020-09-22 at 08 20 56](https://user-images.githubusercontent.com/6448364/93849835-32cc1a00-fcad-11ea-8f17-2b6f04eaa1f7.png)

Reply:
No ale to není chyba kterů bych zanesl tímto PR. Ale vznikla tady: https://github.com/keboola/storage-api-php-client/commit/e59ee05a25c7a3b6d6a3308b7777d9d2b61f4453

Reply:
Tady je špatně odsazení https://github.com/keboola/storage-api-php-client/commit/e59ee05a25c7a3b6d6a3308b7777d9d2b61f4453#diff-2625016b50d68d922257f74801cac29cR5864

Reply:
To je divne, pretoze ja to robim tak, ze si otvorim apiary, kde si dam editor takze by sa mi mala nacitat aktualna verzia a skontrolujem pocet chyb a pak copy&pastnem tie zmeny a kontrolujem ci to cislo nieje vatsie. Hmm tak to nech a ja to potom ked budem upravovat klienta fixnem. Tomas pripravoval nejaky PR na to aby sa to kontrolovalo na CI tak by sa to uz nemuselo stavat